ArduPlane/servos.cpp

```diff
@@ -420,9 +420,9 @@ void Plane::set_servos_controlled(void)
             SRV_Channels::set_output_limit(SRV_Channel::k_throttleLeft, SRV_Channel::SRV_CHANNEL_LIMIT_ZERO_PWM);
             SRV_Channels::set_output_limit(SRV_Channel::k_throttleRight, SRV_Channel::SRV_CHANNEL_LIMIT_ZERO_PWM);
         } else {
```

Comment:
i think this should be a new else if  (arming.arming_required() == AP_Arming::Required::YES_THR_MIN) 
( I have not checked what the enum actually is) and then keep the original else because i think you can be still disarm with AP_ARMING 0, due to failing checks or disarming via GCS. This would output THR_MIn in that case.

Reply:
On AP_ARMING 0 , throttle is free until armed. Nothing is sent. 
This is done [here](https://github.com/ArduPilot/ardupilot/blob/32ebd2080fe2a5cfe9017ee5fa9843f1d49a092f/ArduPlane/servos.cpp#L739)

Reply:
ah, do you not need to add your fix to that switch case then also? (not had a proper look into what is going on)

Reply:
Also, suppress_throttle should handle all cases where throttle must be really PWM min.

Reply:
> ah, do you not need to add your fix to that switch case then also? (not had a proper look into what is going on)

Actually, I've done test by only changing the 1st one and it works fine.
It's a little bit confusing. I think there might be duplicate code that needs clean-up.

Reply:
Ok, I will try this out this evening in SITL and try and understand what is going on better

Reply:
OK, but you should try with safety switch turned on/off while armed/disarmed - I have no idea if SITL supports safety switch simulation -.